packages/fxa-auth-server/test/scripts/recorded-future/check-and-reset.ts

```diff
@@ -21,7 +21,7 @@ const command = [
   'scripts/recorded-future/check-and-reset.ts',
 ];
 
-describe('recorded future credentials search and account reset script', () => {
+describe('#integration - recorded future credentials search and account reset script', () => {
```

Comment:
Marking this as "integration" because the OAuth DB lib tries to connect to MySQL upon import.